source/FilesServer/app/service.py

```diff
@@ -22,10 +22,25 @@ def is_file_exists(
     return True
 
 
-async def s3_get_presigned_url(
-    file_name: str | None, s3_client: boto3.client = Depends(get_s3_client)
-) -> str | None:
+def s3_get_head_object_http_headers(file_name: str) -> dict | None:
```

Comment:
the function name is way too long and i'm not sure this function is even really necessary